litellm/router_strategy/lowest_tpm_rpm_v2.py

```diff
@@ -346,25 +346,19 @@ def _common_checks_available_deployment(
             elif item_tpm is None:
                 continue  # skip if unhealthy deployment
 
-            _deployment_tpm = None
-            if _deployment_tpm is None:
-                _deployment_tpm = _deployment.get("tpm")
-            if _deployment_tpm is None:
-                _deployment_tpm = _deployment.get("litellm_params", {}).get("tpm")
-            if _deployment_tpm is None:
-                _deployment_tpm = _deployment.get("model_info", {}).get("tpm")
-            if _deployment_tpm is None:
-                _deployment_tpm = float("inf")
-
-            _deployment_rpm = None
-            if _deployment_rpm is None:
-                _deployment_rpm = _deployment.get("rpm")
-            if _deployment_rpm is None:
-                _deployment_rpm = _deployment.get("litellm_params", {}).get("rpm")
-            if _deployment_rpm is None:
-                _deployment_rpm = _deployment.get("model_info", {}).get("rpm")
-            if _deployment_rpm is None:
-                _deployment_rpm = float("inf")
+            _deployment_tpm = (
+                        _deployment.get("tpm") or
+                        _deployment.get("litellm_params", {}).get("tpm") or
+                        _deployment.get("model_info", {}).get("tpm") or
+                        float("inf")
+                    )
+
```

Comment:
IMO, this is not the exactly the same as before.

if `_deployment = {"tpm": 0}` then previously  `_deployment_tpm` will be `0` whereas in your case you will look for next place. 

`_deployment_tpm is not None` just checks for None, if you use `or`, then other Falsely values will be affected like, `''`, `[]`, `{}` `0` etc.

If this is only the original intention, only then this should be merged with through testing.

CC: @krrishdholakia